### ISSUE TITLE
How do the DAGs help to understand what a good negative control is?

### ISSUE THREAD

**Issue by PRijnbeek** [mi-erasmusmc/CausalInference#3]
Question raised by Erica.

I think the answer is in [Lipsitch et al. ](https://www.ncbi.nlm.nih.gov/pubmed/20335814):

![screen shot 2018-07-12 at 15 44 02](https://user-images.githubusercontent.com/7252105/42637021-6d204ef6-85ea-11e8-810d-e062fdae7108.png)

"Ideally, a negative control outcome (N) should be an outcome such
that the set of common causes of exposure A and outcome Y should be as identical as
possible to the set of common causes of A and N (Figure 2). To the extent that the set of
unobserved common causes of A and Y overlaps with the set of unobserved common causes
(U) of A and N, we call the negative control outcome N “U-comparable” to Y. If N and Y
are U-comparable outcomes (i.e. with an identical set of common causes that are associated
with A), and assuming that N is not caused by A, an association A-N when analyzed
according to the same procedure used to analyze A-Y would indicate bias in the association
A-Y. If N and Y are perfectly U-comparable and N is not caused by A, then a null finding of
A-N implies that the A-Y association is not likely biased by the pathways examined through
this negative control."

In other words, as the DAG shows, you want the negative control to have exactly the same causal relationships with A and Y except for the causal path from A to Y. In that way you are controlling for "all" measured and unmeasured confounding/bias.

**Comment by ericaVoss** [comment]
Thanks @PRijnbeek, I've read this paper a few times but the terminology makes a bit more sense after our exercise on Friday.  Some notes:

- A = risk factor, actor
- Y = outcome
- L = measured variables
- N = negative control outcome
- U = unobserved common causes
- N should not be caused by A
- N and Y should be U comparable
- You like your L to N to be similar to the L to Y relationship

I'm now going to try to associate Patrick's graphs FIG 2 above.

**Comment by ericaVoss** [comment]
I tried to translate one of Patrick's graphs.  In discussions with Patrick, he told me to simplify the Lipsitch diagram to only the parts that are needed.  

### EXAMPLE 1

- What we are trying to understand is the question of interest, does A cause Y.  
- We use negative controls to help calibrate our methods.  
- In the example, Opioid Use (or by extension Opioid Dependence) are not good negative controls because Infliximab is associated with pain reduction and pain is also associated to opioid use.  There is a causal association between infliximab and opioid use.  

![image](https://user-images.githubusercontent.com/6305439/42966380-f0c0aaaa-8b6a-11e8-949c-f61c661b4ea1.png)

**Comment by ericaVoss** [comment]
### EXAMPLE 2

- In the example, Spider Bite Wound would not be a good negative control for infliximab as there is a relationship between the two.  Infliximab causes a reduced immune system which can cause a spider bite to be come a wound when it otherwise would not.

![image](https://user-images.githubusercontent.com/6305439/42968428-34064d1e-8b71-11e8-9675-2df0e99439bc.png)
